metricbeat/mb/testing/data/data_test.go

```diff
@@ -105,28 +107,35 @@ func getTestdataFiles(t *testing.T, url, module, metricSet, suffix string) {
 
 	for _, f := range files {
 		t.Run(f, func(t *testing.T) {
-			runTest(t, f, module, metricSet, url, suffix)
+			if *moduleFlag != "" {
```

Comment:
Shoudln't this check already happen on line 93? We always want to run all files of a specific module I assume.

Reply:
Yes, you are right. It took me few reads to understand but now I see that we can check this way earlier. Thanks!